tools/DfmHttpService/DfmHttpServer.cs

```diff
@@ -9,18 +9,18 @@ namespace DfmHttpService
 
     internal class DfmHttpServer
     {
-        private const int DefaultPort = 4001;
-        private const string UrlPrefixTemplate = "http://localhost:{0}/";
         private readonly HttpListener _listener = new HttpListener();
         private ManualResetEvent _processing = new ManualResetEvent(true);
         private readonly IHttpHandler _handler;
         private int _status;
 
-        // TODO: make UrlPrefix configurable
-        private static string UrlPrefix => string.Format(UrlPrefixTemplate, DefaultPort);
-
-        public DfmHttpServer(IHttpHandler handler)
+        public DfmHttpServer(IHttpHandler handler, string port)
```

Comment:
It will be checked in extension which will be next PR.

---
In reply to: [108102820](https://github.com/dotnet/docfx/pull/1490#discussion_r108102820) [](ancestors = 108102820)